ee/observe/health_history_handler_test.go

```
@@ -0,0 +1,127 @@
+	r.from = from
+	r.to = to
+	return r.points, nil
+}
+
+func serveHealthHistory(handler *HealthHistoryHandler, appID, query string) *httptest.ResponseRecorder {
+	router := mux.NewRouter()
+	router.HandleFunc(
+		"/api/apps/{APP_ID}/observe/update-health/history",
+		handler.GetUpdateHealthHistoryHandler,
+	).Methods(http.MethodGet)
+	request := httptest.NewRequest(
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)